source/tool/ToolMain.cpp

```diff
@@ -147,7 +155,7 @@ toolMain(
 
     // Run the generator.
     llvm::outs() << "Generating docs...\n";
-    if(! gen->build(config.OutDirectory, *rv, config, R))
+    if(! gen->build(config.OutDirectory, *rv, config, R, Filename))
```

Comment:
This is not right though, because the non-XML generators emit multi-file output

Reply:
does the non-xml have a primary file I can use? I need some target for cmake.